rs_lib/src/lib.rs

```diff
@@ -27,7 +27,7 @@ pub fn generate_bindgen(
 fn inner(name: &str, wasm_bytes: Vec<u8>) -> Result<Output> {
   let mut x = wasm_bindgen_cli_support::Bindgen::new()
     .deno(true)?
-    .weak_refs(true)
+    // .reference_types(true)
```

Comment:
Was this accidentally left in here?

Reply:
```suggestion
```

weak refs is auto detected now. https://github.com/rustwasm/wasm-bindgen/commit/3e469b265464e60645f7eceb5745fa2558aea703